### DISCUSSION TITLE
DAI 2024-2025 - Practical work 1 - Leonard Cseres and Aude Laydu

### DISCUSSION THREAD

**Discussion by leonardcser** [heig-vd-dai-course/heig-vd-dai-course#357] [Show and tell]
### `img2gif` CLI tool

Cet outil permettra de convertir un dossier d'images en format gif avec des paramètres de customisation.

**Example Usage (not definitive):**
img2gif --grayscale --text='Some text'  ./folder output.gif

**Dependency:** https://openimaj.org

**Comment by ludelafo** [comment]
Comme discuté en classe, cela nous semble bon. Merci !

**Comment by leonardcser** [comment]
# Rendu
Updated repo link: https://github.com/HEIGVD-S3-DAI/img2gif.git
Last commit hash: b3878f8c30755ccf10bf01b631775843442100bc
